kosmorrolib/events.py

```diff
@@ -245,6 +247,39 @@ def _search_moon_perigee(start_time: Time, end_time: Time, timezone: int) -> [Ev
     return events
 
 
+def _search_earth_season_change(start_time: Time, end_time: Time, timezone: int) -> [Event]:
+    """Function to find earth season change event.
+
+    **Warning:** this is an internal function, not intended for use by end-developers.
+
+    Will return Summer Solstice and Autumnal Equinox on respectively 2020-06-20 and 2020-09-21:
+
+    >>> season_change = _search_earth_season_change(get_timescale().utc(2020, 6, 20), get_timescale().utc(2020, 6, 21), 0)
+    >>> len(season_change)
+    2
+    >>> oppositions[0].objects[0]
+    'Summer Solstice'
+
+    Will return nothing if there is no season change event in the period of time being calculated:
+
+    >>> _search_earh_season_change(get_timescale().utc(2021, 3, 20), get_timescale().utc(2021, 3, 21), 0)
+    []
+    """
+    
+    events = []
+    t, y = almanac.find_discrete(start_time,end_time,almanac.seasons(get_skf_objects()))
+    if len(t) == 0:
+        return []
+    else:
```

Comment:
This `else` statement can be removed, as the instructions inside will never be executed if the `if` above is true (this is why there is a return :smile:)